src/config.rs

```diff
@@ -17,7 +17,7 @@ pub struct ListenerConfig {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct TermvizConfig {
     pub fixed_frame: String,
-    pub camera_frame: String,
```

Comment:
keep camera frame, as we want it anyways at some point?

Reply:
i would argue that we better get rid of unused params and add them if we need to

Reply:
exactly: I agree with maxi